addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -170,58 +170,14 @@ class EnvironmentMountService extends Service {
         // Get statements for listing and reading study data, respectively
         const statements = s3Policy.Statement;
         s3Prefixes.forEach(prefix => {
+          // eslint-disable-next-line prefer-const
+          let allStatements = this.getAllStatements(s3BucketName, prefix);
+          // eslint-disable-next-line no-unused-vars
```

Comment:
You probably don't need to suppress this warning

Reply:
changed it to const and got rid of the no-unused-vars warning